src/app/pages/admin/search/search-form.component.html

```diff
@@ -1,15 +1,16 @@
 <form [formGroup]="searchForm" (ngSubmit)="onSubmit()">
   <h2>Filter</h2>
-  <strong>Op modules</strong>
-  <ul>
-    <li formArrayName="modules" *ngFor="let module of modules; let i = index">
-      <label>
-        <mat-checkbox [formControlName]="i"></mat-checkbox>
-        {{modules[i].name}}
-      </label>
-    </li>
-  </ul>
-
+  <ng-container *ngIf="isSchool()">
+    <strong>Op modules</strong>
+    <ul>
+      <li formArrayName="modules" *ngFor="let module of modules; let i = index">
+        <label>
+          <mat-checkbox [formControlName]="i"></mat-checkbox>
+          {{modules[i].name}}
```

Comment:
We're iterating over a set, no need to use the index to retrieve the module: `{{module.name}}`

Reply:
fixed